src/main/java/pb360/service/impl/GenerateRestApiServiceImpl.java

```diff
@@ -0,0 +1,70 @@
+package pb360.service.impl;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import org.springframework.stereotype.Service;
+
+import pb360.model.MessageObject;
+import pb360.service.GenerateRestApiService;
+
+@Service
+public final class GenerateRestApiServiceImpl implements GenerateRestApiService {
+
+	@Override
+	public MessageObject getRestApiData(String id) {
+		MessageObject getMessage = new MessageObject();
+		getMessage.setData("Get RestApi data successful" + id);
+		getMessage.setType("GET");
+
+		return getMessage;
+	}
+
+	@Override
+	public MessageObject searchRestApiData() {
+		MessageObject search = new MessageObject();
+		search.setData("Search RestApi data successful");
+		search.setType("GET ALL");
+
+		return search;
+	}
+
+	@Override
+	public MessageObject postRestApiData() {
+		MessageObject post = new MessageObject();
+		post.setData("�generate restapi files successful");
+		post.setType("POST");
+
+		return post;
+
+	}
+
+	@Override
+	public MessageObject patchRestApiData() {
+		MessageObject patch = new MessageObject();
+		patch.setData("updated restapi files successful");
+		patch.setType("PATCH");
+
+		return patch;
+	}
+
+	public MessageObject deleteRestApiData() {
+		MessageObject delete = new MessageObject();
+		delete.setData("delete restapi files successful");
+		delete.setType("DELETE");
+
+		return delete;
+	}
+
+	@Override
+	public MessageObject getCurrentDateTime() {
+		// TODO Auto-generated method stub
```

Comment:
this function is the specific function of this serviceimpl, should not add it on service and keep it only a private function on this

Reply:
- Deleted from service
- Change it to private to access only in serviceimpl class